### DISCUSSION TITLE
Incident in PRD cluster - need advice

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#8186] [Q&A]
Hi

We run a three NODE kafka cluster version 0.28 and kafka 3.1.0 

Due to GKE issue,  one of the Broker is not restarting after an update , and its failing due to MOUNTING TO Persistent disk issue.   

My Q-   KAFKA is still healthy and I can see a lot of traffic still handled by other two NODES.  When will KAFKA stop to work (since we have only 2 NODE and not 3 to elect a leader?)

When I get to fix the broken broker, will ENTITY/Strimzi operator  take care of SYNINCG the broker with up to date data logs from other two NODE To make it operationinal again?

**Comment by scholzj** [comment]
> KAFKA is still healthy and I can see a lot of traffic still handled by other two NODES. When will KAFKA stop to work (since we have only 2 NODE and not 3 to elect a leader?)

That really depends on how you configure your brokers, topics, producers, etc. But yes, when everything is well configured for availability, Kafka can keep going without a single node. But with the most common configurations, losing a second one would break it for producers. So you normally want to fix such a situation as soon as possible since you never know when the next node goes down.

> When I get to fix the broken broker, will ENTITY/Strimzi operator take care of SYNINCG the broker with up to date data logs from other two NODE To make it operationinal again?

TBH, Strimzi is not really involved in this. When the node comes back up, it will reconnect to the remaining two nodes and should start syncing the data on its own - without any intervention from some operator. But if it was shutdown for a long time, it might of course take time.

**Reply by sreejesh-radhakrishnan-db** [reply]
Many thanks for a quick response. 

   So advice from google is restart the pod which is having issue.  If I do a K8s delete of the POD, will Operator be trying to bring the broker POD back up?   If not what is easy way to restart the broker?

**Reply by scholzj** [reply]
Yeah, you can just delete the pod - especially if it is anyway not running.

**Reply by sreejesh-radhakrishnan-db** [reply]
Cool, thank you
